UnityClient/Assets/Scripts/Components/UI/Background.cs

```diff
@@ -0,0 +1,23 @@
+using ROIO;
+using System;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class Background : MonoBehaviour {
```

Comment:
Wouldn't it be better to create an extension for `RawImage` instead of this utility? In the end we're just trying to find the Canvas `RawImage`

Reply:
Do you have an example for this implementation? Sorry, I am a bit rusty in Unity

Reply:
The extension is a plain C# extension like this one https://github.com/guilhermelhr/unityro/blob/develop/UnityClient/Assets/Scripts/Utils/Extensions/CanvasExtensions.cs

And for the Canvas, you would have it like we have the login input field tied to the controller... `[SerializeField] private RawImage background` and then assign the RawImage from the editor to the object

Reply:
If I do have to assign the canvas using the editor, I do not need the extension to find it. Not sure if I get it

Reply:
Anyway, I did some changes in the code to use extensions. Not really what I think you mentioned, but pretty close